tests/test_yaml.py

```diff
@@ -61,13 +63,15 @@ def test_invalid_line():
 
     assert config.as_text() == """line_a: True
 line_b: !2
-        ^
---------
-Basic YAML parsing error
-- could not determine a constructor for the tag '!2'
---------
+#       ^
+# --------
+# Basic YAML parsing error
+# - could not determine a constructor for the tag '!2'
+# --------
 line_c: 8"""
 
+    assert isinstance(config.as_text(), unicode)
```

Comment:
Yeah, I figured that out too, I was just wrangling with tox to try to get local testing for myself working.